src/reports/models/reportEmailSender.js

```diff
@@ -22,6 +22,12 @@ module.exports.sendAggregateReport = async (aggregatedResults, job, emails) => {
         reportId: aggregatedResults.report_id,
         parallelism: aggregatedResults.parallelism
     };
+    if (reportBenchmark.score) {
```

Comment:
what if there is no benchmark? will it have empty section?
if yes need to clean it

Reply:
it will not add it to report, there is if in HTML creator